lib/rubocop/cop/rails/blank.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cops checks for code that can be changed to `blank?`.
+      # Settings:
+      #   NilOrEmpty: true
+      #     Convert checks for `nil` or `empty?` to `blank?`
+      #   NotPresent: true
+      #     Convert usages of not `present?` to `blank?`
+      #   UnlessPresent: true
+      #     Convert usages of `unless` `present?` to `blank?`
+      #
+      # @example
+      #   # bad
+      #   foo.nil? || foo.empty?
+      #   foo == nil || foo.empty?
+      #   nil == foo || foo.size == 0
+      #   something unless foo.present?
+      #   !foo.present?
+      #
+      #   # good
+      #   foo.nil?
+      #   foo.empty?
+      #   foo.blank?
+      #   foo.present?
+      #   !foo.blank?
+      #   something unless foo.blank?
+      #   somethig if foo.blank?
+      #   something if foo.present?
+      class Blank < Cop
+        MSG_NIL_OR_EMPTY = 'Use `blank?` instead of `nil?` or `empty?`.'.freeze
+        MSG_NOT_PRESENT = 'Use `blank?` instead of not `present?`.'.freeze
+        MSG_UNLESS_PRESENT =
+          'Use `if` `blank?` instead of `unless` `present?`.'.freeze
```

Comment:
`if blank?` `unless present?`

I'd actually include the actual receivers in all those message - that would make them way clearer.